app/controllers/oidc_controller.rb

```diff
@@ -65,16 +63,13 @@ def done
     @access_token = access_token
     @id_token = id_token
 
-    # TODO: phantomjs does not download files (https://github.com/ariya/phantomjs/issues/10052), so
-    #       we only set the content-disposition as attachment in production.
-    #
-    # :nocov:
-    if Rails.env.production?
-      response.headers["Content-Disposition"] = "attachment; filename=kubeconfig"
-    end
-    # :nocov:
-
-    render "kubeconfig.erb", layout: false, content_type: "text/yaml"
+    send_data(
+      render_to_string(template: "oidc/kubeconfig.erb", content_type: "text/yaml", layout: false),
+      filename: "kubeconfig"
+    )
```

Comment:
I couldn't test this, but if this is served as an attachment (`Content-Disposition: attachment`), then phantomjs won't be able to fetch the kubeconfig file on automation project (https://github.com/kubic-project/automation/blob/master/velum-bootstrap/spec/features/02-bootstrap-cluster.rb#L158).

Reply:
for that reason i stubbed the `send_data` method in `oidc_controller_spec.rb`

haven't tested it though as i have problems with setting up a devenv

Reply:
But, the stub won't have any effect when running velum-bootstrap? - The file will be served as an attachment, and the bootstrap won't be able to download it?

Reply:
yeah good point, so after all it was intended to just render and not to download, but just in development and test env ... i'll drop the commit again